main.go

```diff
@@ -9,10 +9,17 @@ import (
 
 func main() {
 	myRouter := mux.NewRouter().StrictSlash(true)
-	deviceService := deviceHTTPHandler{service: DeviceService{dao: &inMemoryDeviceDAO{}}}
-	myRouter.HandleFunc("/devices", deviceService.createDevice).Methods(http.MethodPost)
-	myRouter.HandleFunc("/devices/{id}", deviceService.getByID).Methods(http.MethodGet)
-	myRouter.HandleFunc("/devices", deviceService.getAll).Methods(http.MethodGet)
+	deviceService := DeviceService{dao: &inMemoryDeviceDAO{}}
```

Comment:
Please add some log line that says the service was started and is listenning on given port - you will soon see how this improves your life when there are a lot of services.